### ISSUE TITLE
wd claims should also emit claim guids

### ISSUE THREAD

**Issue by nichtich** [maxlath/wikidata-cli#39]
I'd like to get a claim guid for editing `wd claim $subject $property` prints the simplified object value. I'd expect some option, e.g. `wd claims --guid $subject $property`.

**Comment by nichtich** [comment]
In my opinion this should be the default because we can also get claim values by this:

~~~
$ wd d -s Q4115189 | jq .claims.P31
[
  "Q2065736",
  "Q571"
]
~~~

So I'd want

~~~
$ wd claims -j Q4115189 P31
[
  {
    "id": "Q4115189$DF92E189-4E3B-4397-AD41-B23998557E55",
    "value": "Q2065736"
  },
  {
    "id": "Q4115189$02E70CCE-15FE-4FC5-A4C9-A38BF6D7E49B",
    "value": "Q571"
  }
]
~~~

~~~
$ wd claims Q4115189 P31
Q4115189$DF92E189-4E3B-4397-AD41-B23998557E55 Q2065736
Q4115189$02E70CCE-15FE-4FC5-A4C9-A38BF6D7E49B Q571
~~~

By it could also be triggered by an option `--ids`. Anyway the field should be named `id` instead of `guid` because this is the field name in original JSON data.

**Comment by maxlath** [comment]
so since the freshly published [`v5.1.0`](https://github.com/maxlath/wikidata-cli/blob/master/CHANGELOG.md#510---2017-11-09) (thank to [`wikidata-sdk` `v5.3.0`](https://github.com/maxlath/wikidata-sdk/blob/master/CHANGELOG.md#530---2017-11-09)) , you can pass options to the `simplify` function to produce the behavior you describe above:
```sh
wd data --simplify --keep ids Q4115189 | jq .claims.P31
```
or for short
```sh
wd d -sk ids Q4115189 | jq .claims.P31
```

**Comment by maxlath** [comment]
closing this issue as there isn't much activity, please re-open if the proposed solution isn't satisfying
